src/middleware/submissionHandler.js

```diff
@@ -100,6 +100,14 @@ module.exports = (router, resourceName, resourceId) => {
           req.body.data = {};
         }
 
+        if (req.method === 'POST') {
+          const reqHeaders = _.omitBy(req.headers, (value, key) => {
+            return ['x-admin', 'x-jwt-token'].includes(key) || key.match(/auth/gi);
```

Comment:
Should also block "x-token" and "x-remote-token"

Reply:
Done